R/zchunk_LA103.ag_R_C_Y_GLU.R

```diff
@@ -0,0 +1,177 @@
+    # some observations that weren't available in the GTAP-based gridded inventories
+    L103.ag_Prod_Mt_R_C_Y_GLU %>%
+      group_by(GCAM_region_ID, GCAM_commodity, year) %>%
+      summarise(value = sum(value)) %>%
+      ungroup %>%
+      complete(GCAM_region_ID = unique(iso_GCAM_regID$GCAM_region_ID), GCAM_commodity, year, fill = list(value = 0)) %>%
+      arrange(GCAM_commodity, GCAM_region_ID) ->  # so we match old d.s. order
+      L103.ag_Prod_Mt_R_C_Y
+
+
+    # Produce outputs
+    L103.ag_Prod_Mt_R_C_Y_GLU %>%
+      add_title("Crop production by GCAM region / commodity / year / GLU") %>%
+      add_units("Mt") %>%
+      add_comments("Crop production computed based on representative base year, and") %>%
+      add_comments("downscaled to regional production and harvested area in all years by GLU") %>%
+      add_legacy_name("L103.ag_Prod_Mt_R_C_Y_GLU") %>%
+      add_precursors("L101.ag_Prod_Mt_R_C_Y",
+                     "temp-data-inject/L102.ag_Prod_Mt_R_C_GLU") %>%
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      L103.ag_Prod_Mt_R_C_Y_GLU
+
+    L103.ag_Prod_Mt_R_C_Y %>%
+      add_title("Crop production by GCAM region / commodity / year") %>%
+      add_units("Mt") %>%
+      add_comments("Crop production computed based on representative base year, and") %>%
+      add_comments("downscaled to regional production and harvested area in all years") %>%
+      add_legacy_name("L103.ag_Prod_Mt_R_C_Y") %>%
+      same_precursors_as(L103.ag_Prod_Mt_R_C_Y_GLU) %>%
+      add_precursors("common/iso_GCAM_regID") %>%
+      # The sort order in the old dataset is funky, and I'm having trouble
+      # replicating it; but the data are identical. Use the less-stringent sum test.
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_SUM_TEST) ->
+      L103.ag_Prod_Mt_R_C_Y
+
+    L103.ag_HA_bm2_R_C_Y_GLU %>%
+      add_title("Harvested area by GCAM region / commodity / year / GLU") %>%
+      add_units("bm2") %>%
+      add_comments("Agricultural harvested area computed based on representative base year, and") %>%
+      add_comments("downscaled to regional production and harvested area in all years by GLU") %>%
+      add_legacy_name("L103.ag_HA_bm2_R_C_Y_GLU") %>%
+      add_precursors("L101.ag_HA_bm2_R_C_Y",
+                     "temp-data-inject/L102.ag_HA_bm2_R_C_GLU") %>%
+      # The sort order in the old dataset is funky, and I'm having trouble
+      # replicating it; but the data are identical. Use the less-stringent sum test.
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_SUM_TEST) ->
+      L103.ag_HA_bm2_R_C_Y_GLU
+
+    L103.ag_Yield_kgm2_R_C_Y_GLU %>%
+      add_title("Unadjusted agronomic yield by GCAM region / commodity / year / GLU") %>%
+      add_units("kg/m2") %>%
+      add_comments("Agricultural yield computed based on production and harvest area, and") %>%
+      add_comments("downscaled to regional values for all years") %>%
```

Comment:
I don't think the yield was "downscaled". Maybe "using GLU specific production and harvested area"